src/test/scala/algolia/integration/TaskStatusIntegrationTest.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Algolia
+ * http://www.algolia.com/
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package algolia.integration
+
+import algolia.AlgoliaDsl._
+import algolia.AlgoliaTest
+import algolia.responses.{TaskIndexing, TasksMultipleIndex}
+
+import scala.concurrent.Future
+
+class TaskStatusIntegrationTest extends AlgoliaTest {
+
+  after {
+    clearIndices("indexToGetTaskStatus")
+  }
+
+  it("should get the status of a single object task") {
+    val create: Future[TaskIndexing] = client.execute {
+      index into "indexToGetTaskStatus" `object` ObjectToGet("1", "toto")
+    }
+
+    val task = taskShouldBeCreated(create)
+
+    eventually {
```

Comment:
We need `eventually` because it takes some time for a task to be actually published: the very frirst call may indicate the task as "pending" and not "published". 

So `eventually` retries (as defined in [Eventually](http://doc.scalatest.org/3.0.0/index.html#org.scalatest.concurrent.Eventually)) until the block does not throw any exception (in our case a `TestFailedException` if the status is not "published").